tests/session/cmd/test_parallel.py

```diff
@@ -169,3 +171,27 @@ def test_parallel_requires_arg(tox_project: ToxProjectCreator) -> None:
     outcome = tox_project({"tox.ini": ""}).run("p", "-p", "-h")
     outcome.assert_failed()
     assert "argument -p/--parallel: expected one argument" in outcome.err
+
+
+def test_parallel_no_spinner(tox_project: ToxProjectCreator) -> None:
+    """Ensure passing `--parallel-no-spinner` implies `--parallel`."""
+    with mock.patch.object(parallel, "execute") as mocked:
+        tox_project({"tox.ini": ""}).run("p", "--parallel-no-spinner")
+
+    mocked.assert_called_once_with(
+        mock.ANY,
+        max_workers=None,
+        has_spinner=False,
+        live=False,
+    )
+
+    # Test legacy parser
+    with mock.patch.object(parallel, "execute") as mocked:
```

Comment:
Looks like this should be a separate test. One test should test one thing only most of the time.

Reply:
Alright, made the two into their own tests.